modules/drivers/mongo/src/metabase/driver/mongo/parameters.clj

```diff
@@ -64,24 +64,27 @@
     (pr-str x)))
 
 (mu/defn- field->name
-  ([field]
-   (field->name field true))
+  ([field alias]
+   (field->name field alias true))
 
   ([field :- driver-api/schema.metadata.column
+    alias
     pr?]
    ;; for native parameters we serialize and don't need the extra pr
-   (cond-> (mongo.qp/field->name field ".")
+   (cond-> (if (str/blank? alias)
+             (mongo.qp/field->name field ".")
+             alias)
      pr? pr-str)))
```

Comment:
@rileythomp It's not your fault, but this is a bad function. We should not pass in a boolean that determines whether we do extra processing. And it's only used in one place! Here are some options for getting rid of `pr?`:

1. Change the name to `native?` (per the comment).
2. Add quotes to the `format` expressions to be more clear that quotes are added around names in certain places.
3. Add an explicit `pr-str` at the appropriate call sites.
4. Make another function that doesn't pr-str and call that in the one place we don't use it.

It makes sense to improve this function in this PR because you're changing every call site.